bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractFormComponentImpl.java

```diff
@@ -108,6 +110,11 @@ public class AbstractFormComponentImpl extends AbstractComponentImpl implements
     @Nullable
     protected String dorColspan;
 
+    @ScriptVariable(injectionStrategy = InjectionStrategy.OPTIONAL)
+    @JsonIgnore
+    @Nullable
+    protected Style currentStyle;
```

Comment:
Please get ride of @JsonIgnore from here, it should be part of getters only

Reply:
Removed!